api/envoy/data/core/v2alpha/local_reply_body.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package envoy.data.core.v2alpha;
+
+option java_outer_classname = "LocalReplyBodyProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.data.core.v2alpha";
+
+// [#protodoc-title:  core data]
+
+// Wrapper for local reply body data.
```

Comment:
Can we expand this comment to tell why we need this?

Reply:
https://github.com/envoyproxy/envoy/issues/4705

This issue mentions the need to remove the dependency of RapidJSON. The json conversion uses the method of protobuf to convert, so I defined a protobuf for converting the local reply body. I am not sure there are other ways to do the same thing.